data_processors/pipeline/lambdas/orchestrator.py

```diff
@@ -188,75 +188,55 @@ def prepare_germline_jobs(this_batch: Batch, this_batch_run: BatchRun, this_sqr:
     metadata_df = pd.DataFrame(metadata)
     fastq_list_df = pd.DataFrame(fastq_list_rows)
 
-    # iterate through each sample group by rglb
-    for rglb, sample_df in fastq_list_df.groupby("rglb"):
-
-        rgsm = sample_df['rgsm'].unique().item()  # get rgsm which should be the same for all libraries
+    # Add the sample_library_names attribute to fastq_list_df
+    fastq_list_df["sample_library_names"]: fastq_list_df['rgid'].apply(lambda x: x.rsplit('.', 1)[-1])
```

Comment:
Should be assignment equal

```
fastq_list_df["sample_library_names"] = fastq_list_df['rgid'].apply(lambda x: x.rsplit('.', 1)[-1])
```

Reply:
Resolved with c88688c8b5212751a2e3703ffd6ada79ef6271a3